ui/src/views/404/index.jsx

```diff
@@ -1,5 +1,52 @@
 // @flow
+import { colour } from 'pasteup';
+import Logo from 'assets/images/guardian-logo-320.svg';
+import Navigation from './Navigation';
+import Footer from './Footer';
 
-import style from './style.js.scss';
+import {
+    heading,
+    fluidWrap,
+    topbar,
+    subHeadingContainer,
+    subHeading,
+    para,
+} from './style.js.scss';
 
-export default () => <div style={style.message}>404</div>;
+export default () =>
+    <div style={fluidWrap}>
+        <div style={topbar}>
+            <a
+                href="/"
+                style={{
+                    color: '#005689',
+                    textDecoration: 'none',
+                    fontFamily: 'arial',
+                    padding: '8px 10px 7px',
+                    display: 'block',
+                }}>
+                Home
+            </a>
+        </div>
+        <Logo
```

Comment:
I think that if the `<Logo />` contained more than just an svg (png fallback for example) I would agree making a logo component is good. But if it is just an svg then I am inclined to agree with Gustav.

I would personally find it exhausting to have to create a new component every time I wanted to use an svg, and actually I don't think that abstraction is useful. I think its good to be clear I am using an svg for `clock-icon` not just using `ClockIcon` if that makes sense.